web/client/codechecker_client/thrift_helper.py

```diff
@@ -24,17 +24,33 @@
 
 class ThriftClientHelper(object):
 
-    def __init__(self, protocol, host, port, uri, session_token=None):
+    def __init__(self, protocol, host, port, uri, session_token=None,
+                 get_new_token=None):
```

Comment:
Please add a docstring about the get_new_token argument which should be a function which can generate a new token.